src/packages/radio/radio.taro.tsx

```
@@ -136,37 +110,43 @@ export const Radio: FunctionComponent<
+  const renderButton = () => {
+    const buttoncls = classNames(`${classPrefix}-button`, {
+      [`${classPrefix}-button-active`]: checkedStatement,
+      [`${classPrefix}-button-disabled`]: disabled,
+    })
+    return <div className={buttoncls}>{children}</div>
+  }
+  const renderByShape = (shape: RadioShape) => {
+    return shape === 'button' ? renderButton() : renderLabel()
+  }
   const renderRadioItem = () => {
     return renderByShape(context && context.shape ? context.shape : shape)
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)